spring-integration-core/src/test/java/org/springframework/integration/config/InvalidChannelWithMessageStoreParserTests.java

```diff
@@ -17,22 +17,26 @@
 package org.springframework.integration.config;
 
 import org.hamcrest.Matchers;
+import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
-
 import org.springframework.beans.factory.parsing.BeanDefinitionParsingException;
 import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 /**
  * @author Dave Syer
+ * @deprecated replaced by InvalidQueueChannelParserTests and
+ *             InvalidPriorityChannelParserTests
  */
+@Deprecated
 public class InvalidChannelWithMessageStoreParserTests {
```

Comment:
>If we don't need this test at all just remove.
>Since you copy/paste an existing tests to a new one class, 
>pleas, show respect and leave Dave Syer's name there as well

That's why I did use that annotation, ok I will remove that test class and add the author note in the new version